apps/build.gradle

```diff
@@ -31,7 +31,7 @@ task format(type: YarnTask, dependsOn: yarn_install) {
     }
 }
 
-task checkFormat(type: YarnTask, dependsOn: format) {
```

Comment:
the idea was to sync the check ( fist format , then check ( on CI only do check ) , why would we change this ?

Reply:
did you check if removing the sync does not break the local gradle build ( check running before format) , this pr mostly fixes issue already fixed ( before update with master was done )